torch/_torch_docs.py

```diff
@@ -8292,25 +8292,30 @@ def merge_dicts(*dicts):
 svd(input, some=True, compute_uv=True, *, out=None) -> (Tensor, Tensor, Tensor)
 
 Computes the singular value decomposition of either a matrix or batch of
```

Comment:
"of a matrix or the decompositions of a batch of matrices input."

Otherwise this sentence would say there's one SVD for the batch

Reply:
I'll apply the template you suggested here https://github.com/pytorch/pytorch/pull/56265#discussion_r615376640